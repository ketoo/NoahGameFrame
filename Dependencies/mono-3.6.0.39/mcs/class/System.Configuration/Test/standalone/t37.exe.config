<configuration>
<connectionStrings>
        <remove name="LocalSqlServer" />
        <add name="LocalSqlServer"
                connectionString="data source=127.0.0.1; Integrated Security=SSPI" />
        <add name="AccessFileName"
                connectionString="~/Data/ASPNetDB.mdb" />
        <remove name="AccessFileName" />
</connectionStrings>

<appSettings>
	<add key="hithere" value="Hello World!"/>
	<remove key="hithere" />
</appSettings> 

<configProtectedData defaultProvider="MyRsaProtectedConfigurationProvider">
    <providers>
      <clear />
      <add name="MyRsaProtectedConfigurationProvider"
	type="System.Configuration.RsaProtectedConfigurationProvider, System.Configuration, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"
        keyContainerName="MyKeys" 
        useMachineContainer="true" useOAEP="true" />
      <remove name="MyRsaProtectedConfigurationProvider" />
    </providers>
  </configProtectedData> 

</configuration>
