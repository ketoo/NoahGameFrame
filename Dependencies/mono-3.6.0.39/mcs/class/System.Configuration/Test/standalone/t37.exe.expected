connStrings[LocalSqlServer] = data source=127.0.0.1; Integrated Security=SSPI
connStrings[AccessFileName] = null
sect.Settings[hithere] = null
rsa = null
